public/app/features/alerting/unified/components/AlertManagerPicker.tsx

```diff
@@ -30,20 +33,33 @@ export const AlertManagerPicker: FC<Props> = ({ onChange, current, disabled = fa
     ];
   }, []);
 
+  // no need to show the picker if there's only one option
+  if (options.length === 1) {
+    return null;
+  }
+
   return (
-    <Select
-      disabled={disabled}
-      width={29}
-      className="ds-picker select-container"
-      isMulti={false}
-      isClearable={false}
-      backspaceRemovesValue={false}
-      onChange={(value) => value.value && onChange(value.value)}
-      options={options}
-      maxMenuHeight={500}
-      noOptionsMessage="No datasources found"
-      value={current}
-      getOptionLabel={(o) => o.label}
-    />
+    <Field className={styles.field} label={disabled ? 'Alert manager' : 'Choose alert manager'} disabled={disabled}>
+      <Select
+        disabled={disabled}
+        width={29}
+        className="ds-picker select-container"
+        isMulti={false}
```

Comment:
nit: I believe this defaults to `false`.

Reply:
removed